kubernetes/overlays/notifier-seeder/dev-refactor/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  KAFKA_CLIENT_ID: "dev-pagopa-interop-notifier-seeder"
```

non-blocking idea: could we use the env var `STAGE` here?
it would allow to have the same manifest for all envs and avoid forgetting to change it on deployment.
```suggestion
  KAFKA_CLIENT_ID: "{{STAGE}}-pagopa-interop-notifier-seeder"
```

---

Nice catch, thank you! 
Btw I don't like this client ID format, but that's another topic (no pun intended 🤣 )